### PR TITLE
New region variables to override provider regions

### PR DESCRIPTION
**Pull request by syedrakib**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -164,6 +164,18 @@ variable "authorized_networks_read_replica" {
   default = []
 }
 
+variable "region_master_instance" {
+  description = "The region to launch the master instance in. Defaults to the Google provider's region if nothing is specified here. See https://cloud.google.com/compute/docs/regions-zones"
+  type        = string
+  default     = ""
+}
+
+variable "region_read_replica" {
```
Comment (itsmunim) [inline variables.tf:173]: Do we need to give this as an option? Normally read replicas should be in colocated in zones in the same region as master is. Region wise separation most of the time happen when it's a DR setup. Here it should always provision replicas in the region that's selected for master.

**Review by itsmunim — DISMISSED** [review]
LGTM. Except the comment I have left.

File: variables.tf (hunk 1 above)
Comment (syedrakib) [reply variables.tf:173]: We have a use case with one of our projects. That is specifically why this PR was created. Will message privately

If anyone leaves this blank (which is the default), it uses the master region as usual as you pointed out.

**Review by itsmunim — APPROVED** [review]
LGTM

**Pull request merged**